pkg/agents/team6/strategies.go

```diff
@@ -1,24 +1,93 @@
 package team6
 
+import (
+	"math"
+	"math/rand"
+
+	"github.com/SOMAS2021/SOMAS2021/pkg/utils/globalTypes/health"
+)
+
 type thresholdData struct {
-	satisficeThresh float64
-	satisfyThresh   float64
-	maxIntake       float64
+	maxIntake float64
 }
 
+type levelsData struct {
+	strongLevel  float64
+	healthyLevel float64
+	weakLevel    float64
+	critLevel    float64
+}
+
+var daysInCritical int = 0
+var foodTakeDay int
```

Comment:
```suggestion
daysInCritical := 0
foodTakeDay := 0
```

Reply:
We need these both outside the function, as they get updated inside the function. I think you can only use short assign inside of functions?

Reply:
nonononono no global variables